dhall-docs/src/Dhall/Docs.hs

```diff
@@ -0,0 +1,70 @@
+{-| This module contains the top level and options parsing of the @dhall-docs@
+    executable
+-}
+{-# LANGUAGE OverloadedStrings #-}
+
+module Dhall.Docs
+    ( -- * Options
+      Options(..)
+    , GenerationStrategy(..)
+    , parserInfoOptions
+    , parseOptions
+
+      -- * Execution
+    , main
+    ) where
+
+import Data.Semigroup ((<>))
+
+import qualified Options.Applicative
```

Comment:
Perhaps import `Parser` unqualified.  I usually try to import types unqualified when there are no conflicts